docs/source/daml/intro/11_Testing.rst

```diff
@@ -0,0 +1,283 @@
+
+We now want establish more relationships and issue assets. We could do that using Navigator, but it would be some work.
+
+DAML Script with Inputs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You may be tempted to run the script ``Test.Intro.Asset.MultiAsset.testMultiTrade``, but beware! That script calls ``tradeSetup``, which allocates parties. If we ran that script, we would run an entirely new setup, and then multi-leg trade on the contracts resulting from the second ``tradeSetup``. Try it out if you want to see that in action.
+
+Fortunately, DAML Script can take inputs in JSON format, and all the ``setupRelationship`` and ``issueAsset`` scripts are set up for that by taking ``Relationship`` and ``Asset`` record value as input. In order to use that feature, we first have to get our hands on a new party. Run ``daml ledger allocate-party CHF_Bank`` to allocate the party. You can list all known parties using ``daml ledger list-parties``.
+
+.. code-block:: none
+
+  Listing parties at localhost:6865
+  PartyDetails {party = 'party-27089832', displayName = "Alice", isLocal = True}
+  PartyDetails {party = 'party-6ccc1f1e', displayName = "Bob", isLocal = True}
+  PartyDetails {party = 'party-084d5d8e', displayName = "USD_Bank", isLocal = True}
+  PartyDetails {party = 'party-cee70c7b', displayName = "EUR_Bank", isLocal = True}
+  PartyDetails {party = 'CHF_Bank', displayName = "CHF_Bank", isLocal = True}
+
+Now we can establish a relationship between Alice and the new party. The JSON input for a ``Relationship`` looks as you might expect:
+
+.. code-block:: json
+
+  {
+    "issuer" : "CHF_Bank",
+    "owner" : "party-27089832"
+  }
+
+To execute the DAML Script, run the below command, replacing the party IDs.
+
+.. code-block:: none
+
+  daml script --dar multi-trade.dar --ledger-host localhost --ledger-port 6865 --script-name Test.Intro.Asset.MultiTrade:setupRelationship --input-file <(echo '{"issuer" : "CHF_Bank", "owner" : "party-27089832"}')
+
+Note that the ``--input-file`` flag expects a file. The ``<(..)`` is a bit of unix shell magic to provide the file contents inline. If it doesn't work on your shell, put the JSON into a file ``relationship.json``, and supply that instead. For more information on this, please refer to the :doc:`DAML Script </daml-script/index>` and :doc:`JSON Format </json-api/lf-value-specification>` docs.
+
+If you still have the Navigator open, you'll see a new ``AssetHolder`` for Alice appear as you run the script. That worked, but it would be cumbersome if we wanted to step through the entire multi-leg trade script this way.
+
+DAML REPL
```

Comment:
Same thing here, the docs for DAML REPL should be pretty approachable. Why can’t we mention DAML REPL with a link to the docs for that instead of replicating half of the docs here?